mne/__init__.py

```diff
@@ -98,6 +98,7 @@
 from . import viz
 from . import decoding
 from . import realtime
+from . import testing
```

Comment:
@agramfort I think we should gradually move our testing-specific code here. For now I've only moved `assert_meg_snr`, which is now used by both `mne/preprocessing/tests/test_maxwell.py` and `mne/tests/test_epochs.py`. Should we name it `mne._testing` to make it private?